### PR TITLE
Feature/logo

### PR DESCRIPTION
**Pull request by Shama95**
added a logo,
added a css file as well.
removed all the junk.

### CODE REVIEW

**Comment by AvivLevi815** [comment]
Looks great, Mabey add some descriptive comments.

**Comment by Shama95** [comment]
i added the comments and changed the size of the image. also i added addir's datepicker include.

File: src/Components/App.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ import Otzar from "./Otzar";
 import Loading from "./Loading";
 
 const App = () => {
-  return (
+  return (//in the app component we use the route component to chose the component being used according to the path.
```
Comment (AvivLevi815) [inline src/Components/App.js]: pretty long, how about:
"use Route to navigate based on the path"?

File: src/Components/Header.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import Link from './Link'
 
 const Header =()=>{
-    return (
+    return (//this component changes each url according to what the user presses in the header.
```
Comment (AvivLevi815) [inline src/Components/Header.js]: pretty long, how about:
"Use Link to navigate"?

File: src/Components/Loading.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import Logo from "../images/sachar.jpg";
 import '../css/app.css';
 
 const Loading=()=>{
-    return (
+    return (//this component is the main page component, it uses the css file to center the message and the image.
```
Comment (AvivLevi815) [inline src/Components/Loading.js:6]: Consider splitting the CSS part to a different comment

File: src/css/app.css [hunk 4]
```diff
@@ -1,7 +1,6 @@
 .App {
-    text-align: center;
+    text-align: center;/*aligning of the text in the center*/
```
Comment (AvivLevi815) [inline src/css/app.css]: I don't think this comment is needed

File: src/css/app.css [hunk 5]
```diff
@@ -1,7 +1,6 @@
...
   }
   .App-logo {
-    height: 40vmin;
-    pointer-events: none;
+    height: 70vmin;/*decides the size of the image*/
```
Comment (AvivLevi815) [inline src/css/app.css]: size? not height?
if you mean size - consider move the commebt next to .App-logo

**Review by AvivLevi815 — COMMENTED** [review]
I wrote some suggestions about the comments/

File: src/Components/Loading.js (hunk 3 above)
Comment (Shama95) [reply src/Components/Loading.js:6]: i didn't understand the comment,
the css is in a different file.

**Review by AvivLevi815 — APPROVED** [review]
looks grate

**Comment by AvivLevi815** [comment]
merged.

**Pull request merged**